examples/podsecuritypolicy/rbac/bindings.yaml

```diff
@@ -31,7 +31,8 @@ roleRef:
    kind: ClusterRole
    name: restricted-psp-user
 ---
-# edit grants edit role to system:authenticated.
+# edit grants edit role to the groups
+# restricted and privileged.
```

Comment:
Good catch!
But I think the group names are:
privileged-psp-users and restricted-psp-users